modules/gapi/samples/face_detection.cpp

```diff
@@ -56,45 +56,45 @@ namespace {
 #define NUM_PTS 5
 
 struct BBox {
-    double x1;
-    double y1;
-    double x2;
-    double y2;
+    int x1;
+    int y1;
+    int x2;
+    int y2;
 
-    cv::Rect getRect() const { return cv::Rect(static_cast<int>(x1),
-                                               static_cast<int>(y1),
-                                               static_cast<int>(x2 - x1),
-                                               static_cast<int>(y2 - y1)); }
+    cv::Rect getRect() const { return cv::Rect((x1),
```

Comment:
I think, this cv::Rect has a lot of brackets.

Also cv::Rect can be constructed by two cv::Points, but it is taste.

Reply:
Done